src/clib/lib/job_queue/lsf_driver.cpp

```diff
@@ -402,6 +417,117 @@ static void run_bjobs(lsf_driver_type *driver, char *output_file) {
     }
 }
 
+static char *strip(const char *src) {
+    char *target{};
+    size_t strip_length = 0;
+    size_t end_index = strlen(src) - 1;
+    while (end_index >= 0 && src[end_index] == ' ')
+        end_index--;
+
+    if (end_index >= 0) {
+        int start_index = 0;
+        while (src[start_index] == ' ')
+            start_index++;
+        strip_length = end_index - start_index + 1;
+        target = (char *)calloc(strip_length + 1, sizeof *target);
+        memcpy(target, &src[start_index], strip_length);
+        CHECK_ALLOC(target);
```

Comment:
I think these lines should be swapped as you are checking if it was malloced correctly after using it.
```suggestion
        CHECK_ALLOC(target);
        memcpy(target, &src[start_index], strip_length);
```
Or you can just avoid checking `CHECK_ALLOC` and let it segfault. (I prefer this solution)